src/fsharp/DependencyManager.Integration.fs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+/// Helper members to integrate DependencyManagers into F# codebase
+module internal Microsoft.FSharp.Compiler.DependencyManagerIntegration
+
+open System
+open System.IO
+open Microsoft.FSharp.Compiler.ErrorLogger
+
+// NOTE: this contains mostly members whose intents are :
+// * to keep ReferenceLoading.PaketHandler usable outside of F# (so it can be used in scriptcs & others)
+// * to minimize footprint of integration in fsi/CompileOps
+
+/// hardcoded to net461 as we don't have fsi on netcore
+let targetFramework = "net461"
+
+
+type IDependencyManagerProvider =
+    inherit System.IDisposable
+    abstract Name : string
+    abstract ToolName: string
+    abstract Key: string
+    abstract ResolveDependencies : string * string * string * string seq -> string * string list
```

Comment:
In the return type, we should consider replacing the list with an array so the type signature remains F# agnostic.